rabbitmq-broker.yaml

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 The Knative Authors
```

Comment:
same here with the yaml file, looks like it's a combination of the files in the config/broker and we don't really want them in the root of the project.